.github/workflows/self-scheduled-amd-caller.yml

```diff
@@ -3,23 +3,12 @@ name: Self-hosted runner (AMD scheduled CI caller)
 on:
   schedule:
     - cron: "17 2 * * *"
-  push:
-    branches:
-      - run_amd_scheduled_ci_caller*
 
 jobs:
-  run_amd_ci_mi210:
-    name: AMD mi210
-    if: (cancelled() != true) && ((github.event_name == 'schedule') || ((github.event_name == 'push') && startsWith(github.ref_name, 'run_amd_scheduled_ci_caller')))
-    uses: ./.github/workflows/self-scheduled-amd.yml
-    with:
-      gpu_flavor: mi210
-    secrets: inherit
-
-  run_amd_ci_mi250:
-    name: AMD mi250
-    if: (cancelled() != true) && ((github.event_name == 'schedule') || ((github.event_name == 'push') && startsWith(github.ref_name, 'run_amd_scheduled_ci_caller')))
-    uses: ./.github/workflows/self-scheduled-amd.yml
-    with:
-      gpu_flavor: mi250
-    secrets: inherit
+  run_scheduled_amd_ci:
```

Comment:
trigger the 2 new workflow file `mi210-caller` and `mi250-caller` via workflow run event. They will run as 2 independent workflow runs.